### PR TITLE
Feature/sem 059 bis

### PR DESCRIPTION
**Pull request by Ditmar**

### CODE REVIEW

**Comment by DanX3N0** [comment]
FIGMA:
image

TICKET:
image

Nice PR, but it does not follow the colors proposed in the figma, you could have 2 variations, which can be seen in the figma.

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "@astrojs/react": "3.3.0",
     "@types/react": "18.2.79",
     "@types/react-dom": "18.2.25",
-    "astro": "4.6.3",
+    "astro": "^4.6.3",
```
Comment (Ditmar) [inline package.json:31]: don't use the carret

Comment (Ditmar) [reply package.json:31]: Ok I will modified this part of this pr, please in the next presentation be careful with this part

**Review by Ditmar — COMMENTED** [review]
Please check the mobile responsive 

image

**Review by Ditmar — COMMENTED** [review]
Nice work

**Pull request merged**